crypto/err/err.c

```diff
@@ -583,8 +504,13 @@ void ERR_error_string_n(unsigned long e, char *buf, size_t len)
         ls = lsbuf;
     }
 
-    rs = ERR_reason_error_string(e);
     r = ERR_GET_REASON(e);
+    if (ERR_SYSTEM_ERROR(e)) {
```

Comment:
A one-line comment would be helpful.

Reply:
Done...  although it became more than one line.